j2lint/cli.py

```diff
@@ -145,21 +195,19 @@ def run(args=None):
 
     # Collect the rules from the configuration
     collection = RulesCollection(options.verbose)
-    for rulesdir in options.rules_dir:
+    for rules_dir in options.rules_dir:
         collection.extend(RulesCollection.create_from_directory(
-            rulesdir, options.ignore, options.warn))
+            rules_dir, options.ignore, options.warn))
 
     # List lint rules
     if options.list:
-        rules = "Jinja2 lint rules\n{}\n".format(collection)
+        rules = f"Jinja2 lint rules\n{collection}\n"
         print(rules)
         logger.debug(rules)
         return 0
 
     # Version of j2lint
-    if options.version:
-        print(f"Jinja2-Linter Version {VERSION}")
-        return 0
+    print_j2lint_version(options)
```

Comment:
This is changing the behavior here

Before:
if you give the version option then you print the version and exit

After:
you always call the version function but do not exit

Please revert this back and remove the method above. let's limit the refactoring.

Reply:
This is not resolved, the behavior is changed and it should not. Please change it back to:

```suggestion
    if options.version:
        print(f"Jinja2-Linter Version {VERSION}")
        return 0
```